<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoView</class>
 <widget class="QWidget" name="VideoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>837</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VideoView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">


/*顶部样式*/
#head {
background-color: rgb(85, 170, 255);
}
#logo
{
	image: url(:/new/prefix1/image/app.ico);
}
#btnMin{
	border-image: url(:/new/prefix1/image/min.png);
}
#btnMax{
	border-image: url(:/new/prefix1/image/max.png);
}
#btnNormal{
	
	border-image: url(:/new/prefix1/image/normal.png);
}
#btnClose{
	border-image: url(:/new/prefix1/image/close.png);
}
#body_widget{
	background-color: rgb(33, 33, 33);
}
#left_widget{
	background-color: #252525;
}
#cames_widget VideoWidget {
	background-color: #1e1e1e;
}

#left_widget  QPushButton:hover
 { 
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 102, 184, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(0, 102, 184, 255));
border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
#left_widget  QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 50, 150, 255), stop:0.495 rgba(0, 102, 184, 255), stop:0.505 rgba(0, 102, 184, 255), stop:1 rgba(0, 50, 150, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }</string>
  </property>
  <widget class="QWidget" name="head" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="logo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="minMax" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMax">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMin">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNormal">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>btnNormal</zorder>
    <zorder>btnClose</zorder>
    <zorder>btnMin</zorder>
    <zorder>btnMax</zorder>
   </widget>
  </widget>
  <widget class="QWidget" name="body_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>771</width>
     <height>491</height>
    </rect>
   </property>
   <widget class="QWidget" name="left_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>200</width>
      <height>256</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnAdd">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnEdit">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnDelete">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QListWidget" name="videoList">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="cames_widget" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>351</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnMin</sender>
   <signal>clicked()</signal>
   <receiver>VideoView</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>VideoView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
